.github/workflows/check-cuda-formatted.yml

```
@@ -0,0 +1,36 @@
+# This file was copied mostly from check-maintainers-sorted.yaml
+name: Check that CUDA files are formatted
+
+on:
+  pull_request_target:
+    paths:
+      - pkgs/development/cuda-modules/**.nix
+      - pkgs/test/cuda/**.nix
+      - pkgs/top-level/cuda-packages.nix
```

```suggestion
  pull_request_target:
```

We can then run it for all PRs

---

Do we want to run the check for all PRs at this point? Maybe we maintain (in the workflow definition) a list of subsystems that explicitly opt-in into enforcing the experimental style

---

I'd say maybe yes, but also this should probably be factored out into a separate PR then